aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -1265,6 +1265,282 @@ const auto erfcx_string = jiterator_stringify(
   }
 ); // erfcx_string
 
+const auto fresnel_integral_c_string = jiterator_stringify(
+    template<typename T>
+    T fresnel_integral_c(T x) {
+        static const T CN[] = {
```

Comment:
Should we add a comment in the future that we may want different coefficients for float vs. double?

Reply:
Good catch. I will go ahead and add them now since Steve Moshier did the work of pre-computing 32-bit coefficients.